proto/orc_proto.proto

```diff
@@ -74,6 +74,13 @@ message BinaryStatistics {
   optional sint64 sum = 1;
 }
 
+// ORC-398: Statistics for list and map
+message CollectionStatistics {
+  optional uint64 minCollectionLength = 1;
+  optional uint64 maxCollectionLength = 2;
+  optional uint64 childCount = 3;
```

Comment:
Can we avoid adding new fields in the file meta by aggregating the stats directly from child col stats?

Reply:
Not sure what you mean by 

> aggregating the stats directly from child col stats

We could use IntegerStatistics but then that would be misleading, this specifically deals with Maps and Lists and the idea is to have min, max, and sum of the number of children for Map and List.

Reply:
I think we are better leaving it separate.